### PR TITLE
Fix a typo, and make a couple minor improvements to README.

### PR DESCRIPTION
**Pull request by jguiditta**
Signed-off-by: Jason Guiditta <jguiditt@redhat.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,15 +52,17 @@ for us, which you can activate via::
 
 See the `pipenv documentation <https://docs.pipenv.org/>`_ for more detail.
 
+Alternatively, you can used a standard python virtualenv if you prefer.
```
Comment (jpichon) [inline README.rst:55]: used -> use?

Comment (jguiditta) [reply README.rst:55]: Geez, I read this like 3 times before submitting it and STILL had a typo!  Thanks, I will fix it.

**Review by fuzzball81 — APPROVED** [review]

**Review by jpichon — APPROVED** [review]
Haha, that's always the way!

**Comment by eggmaster** [comment]
LMTM (looks merged to me ;)) but also LGTM

**Pull request merged**